com.valvesoftware.Steam.yml

```diff
@@ -89,12 +89,6 @@ cleanup:
   - /lib/*.la
   - /lib32/*.a
   - /lib32/*.la
-  - /include
-  - /lib/pkgconfig
-  - /lib32/pkgconfig
```

Comment:
Hm, why are these development files needed?

Reply:
For building compatibility tools as extensions. We don't need development files from all modules, though.

Reply:
Hmm, okay. It makes this entire app extension thing quite suboptimal. I wonder if I could create app Sdk extension that could be used....

Reply:
It would be nice if flatpak-builder could produce some kind of SDK-like bundle of the app with development files kept in place.

Reply:
I was mostly thinking if I could roll an app extension in this manifest and then you could map that as Sdk extension when building your app extension.

Reply:
Maybe we need to do that as follow-up. This is atm blocking things.